connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java

```diff
@@ -651,6 +651,10 @@ public Config describeTopicConfig(String topic) {
      * @param partitions the topic partitions
      * @return the map of offset for each topic partition, or an empty map if the supplied partitions
      *         are null or empty
+     * @throws UnsupportedVersionException if the admin client cannot read end offsets
+     * @throws TimeoutException if the offset metadata could not be fetched before the amount of time allocated
+     *         by {@code request.timeout.ms} expires, and this call can be retried
+     * @throws LeaderNotAvailableException if the leader was not available and this call can be retried
      * @throws RetriableException if a retriable error occurs, the operation takes too long, or the
```

Comment:
`the operation takes too long` 
^^^^^^^^^^^^^^^^^^^^^^^ this message gets invalid since `TimeoutException` is not wrapped to `RetriableException` anymore.